spec/models/user_spec.rb

```diff
@@ -1,5 +1,15 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
```

Comment:
Please test for last name as well -- using shoulda-matchers will make this easier :)